<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT3DReconstructionClass</class>
 <widget class="QMainWindow" name="QT3DReconstructionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>J3D可视化平台</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>J3D_GUI.ico</normaloff>J3D_GUI.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_engine">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Engine</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前程序运行状况：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_process">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>710</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>等待任务</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>720</y>
      <width>671</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="mvsviewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1341</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_camera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>渲染相机位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pointcloud">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>渲染点云</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mesh">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>渲染网格</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_texture">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>渲染纹理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pointplus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点大小+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pointsub">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点大小-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pointnumplus">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>渲染点数量+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pointnumsub">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>渲染点数量-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_export">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输出为其他格式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_viewportplus">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>相机视角+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_viewportsub">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>相机视角-</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ply</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>obj</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>osgb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gltf</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>分步三维重建</string>
    </property>
    <widget class="QMenu" name="menuSFM_Engine">
     <property name="title">
      <string>SFM引擎</string>
     </property>
     <addaction name="actionMatchFeature"/>
     <addaction name="actionSFM"/>
    </widget>
    <widget class="QMenu" name="menuMVS_Engine">
     <property name="title">
      <string>MVS引擎</string>
     </property>
     <addaction name="actionDenseCloud"/>
     <addaction name="action_reconstrctMesh"/>
     <addaction name="action"/>
    </widget>
    <addaction name="menuSFM_Engine"/>
    <addaction name="menuMVS_Engine"/>
    <addaction name="action_import_BlocksExchange"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_mvs_file"/>
    <addaction name="action_2"/>
    <addaction name="action_addSensorWidth"/>
    <addaction name="action_coor"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>全自动三维重建</string>
    </property>
    <addaction name="action_fullauto"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionMatchFeature">
   <property name="text">
    <string>特征匹配</string>
   </property>
  </action>
  <action name="actionSFM">
   <property name="text">
    <string>SFM及稀疏点云生成</string>
   </property>
  </action>
  <action name="actionDenseCloud">
   <property name="text">
    <string>密集点云生成</string>
   </property>
  </action>
  <action name="action_addSensorWidth">
   <property name="text">
    <string>添加相机传感器参数</string>
   </property>
  </action>
  <action name="action_reconstrctMesh">
   <property name="text">
    <string>三角网模型重建</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>模型纹理映射</string>
   </property>
  </action>
  <action name="actionopen_mvs_file">
   <property name="text">
    <string>打开模型</string>
   </property>
  </action>
  <action name="action_fullauto">
   <property name="text">
    <string>全自动工程配置</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开模型（兼容模式）</string>
   </property>
  </action>
  <action name="action_coor">
   <property name="text">
    <string>坐标映射</string>
   </property>
  </action>
  <action name="action_import_BlocksExchange">
   <property name="text">
    <string>导入其他SFM数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mvsviewer</class>
   <extends>QWidget</extends>
   <header>mvsviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>btn_onclick()</slot>
 </slots>
</ui>
